forkable/forkdb.go

```diff
@@ -239,12 +240,6 @@ func (f *ForkDB) ReversibleSegment(upToBlock bstream.BlockRef) (blocks []*Block)
 	curNum := upToBlock.Num()
 
 	for {
-		if curNum == 1 {
```

Comment:
Shouldn't we have some protocol specific block here? The `cur > 2` seems also protocol specific thingy.